### DISCUSSION TITLE
Connect wallet button

### DISCUSSION THREAD

**Discussion by Atoo35** [web3ui/web3uikit#645] [Q&A] [answered]
Hi all, i am new to using this ui kit for web3 development. I was able to use a lot of other components but i am not able to figure out how to use the Connect Wallet button if my dapp has no relation to Moralis as of now. 
I understand there's a property to disable it but the compiler always throws an error that Moralis provider does not exist. 
Can someone please help me here?

**Comment by AbhinavMV** [comment, marked as answer]
Hey @Atoo35 
Sorry for the delay in response. Missed notification for this. 😅 
As of now, you need MoralisProvider to use the component. the property stops the component from updating moralis tables whenever user authenticates through the component.
